google-http-client/src/main/java/com/google/api/client/util/Base64.java

```diff
@@ -14,19 +14,13 @@
 
 package com.google.api.client.util;
 
+import com.google.common.io.BaseEncoding;
+
 /**
- * Proxy for version 1.6 (or newer) of the Apache Commons Codec
- * {@link org.apache.commons.codec.binary.Base64} implementation.
- *
- * <p>
- * This is needed in order to support platforms like Android which already include an older version
- * of the Apache Commons Codec (Android includes version 1.3). To avoid a dependency library
- * conflict, this library includes a reduced private copy of version 1.6 (or newer) of the Apache
- * Commons Codec (using a tool like jarjar).
- * </p>
+ * Convenient and now anachronistic proxy to 3rd party Base64 codec.
```

Comment:
Maybe instead: "Proxy for base 64 encoding/decoding which matches the Base64 interface in Apache Commons (for historical reasons)."